templates/zerver/help/reply-to-message.md

```diff
@@ -0,0 +1,57 @@
+# Replying to a Message
+## There are three ways to reply to a message in Zulip:
+### First Way:
+To reply to a message, click on the message.
+A messaging box will appear with the **Stream** (outlined in red below)
+and **Topic** (outlined in green below) fields
+auto filled to match the message you clicked on.
+
+![reply messaging box](/static/images/help/reply-messaging-box.png)
+
+You can type your reply in the **Compose your message here...** field.
+Learn about formatting messages [here](/help/format-your-message-using-markdown).
+
+### Second Way:
+Mouse over the message that you want to reply to.
+Three icons will appear at the far right side of the message.
+
+![hover icons](/static/images/help/message-hover.png)
+
+Click on the chevron (![chevron](/static/images/help/chevron.png)) icon.
+This menu will appear:
+
+![menu](/static/images/help/menu-chevron-message.png)
+
+Click on **Reply**. A messaging box will appear with the **Stream** (outlined in red below)
+and **Topic** (outlined in green below) fields
+auto filled to match the message you selected.
+
+![reply messaging box](/static/images/help/reply-messaging-box.png)
+
+You can then type your reply in the **Compose your message here...** field.
+
+### Third Way:
+On the Zulip home screen, you will notice a blue box around one of the
+messages on your screen.
+
+![blue box](/static/images/help/blue-box-message.png)
+
+This box indicates which message is currently highlighted and
```

Comment:
This sentence is a bit long, maybe you can split it into "This box indicates which message is currently highlighted" and "it is found on the home screen, in all streams, in your private
 +messages, in your starred messages, and in your @-mentions."